.github/workflows/pack.yml

```diff
@@ -0,0 +1,45 @@
+name: Pack extensions
+
+on:
+  push:
+    paths-ignore:
+      - README.md
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [21.x]
+    container: archlinux:base-devel
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Cache
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: node_modules
+
+      - name: Prepare
+        run: |
+          pacman -Sy
+          pacman -S --noconfirm base util-linux util-linux-libs gnome-shell gtk3 libadwaita gtk4 glib2 gobject-introspection gettext just
```

Comment:
I just tested in a fresh `archlinux:latest` container, and none of those dependencies are needed (except for `just`)